### PR TITLE
refactor login to be a minimal component

### PR DESCRIPTION
**Pull request by FlutterJoey**

### CODE REVIEW

**Comment by stein155** [comment]
Het scherm wachtwoord vergeten is niet meer opgenomen in deze branch, deze zou ook in de plug-in meegenomen worden toch?

**Comment by FlutterJoey** [comment]
Yes, that is correct. I will add those tomorrow as well as fix the conflicts relating to this merge.

I will update this PR before 12 AM tomorrow.

Also, please use english 😉

File: lib/src/config/login_options.dart [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Widget _createRequestForgotPasswordButton(
+  BuildContext context,
+  OptionalAsyncCallback onPressed,
+  bool disabled,
+) {
+  return Opacity(
+    opacity: disabled ? 0.5 : 1.0,
+    child: TextButton(
+      onPressed: onPressed,
+      child: const Text('Forgot password?'),
```
Comment (stein155) [inline lib/src/config/login_options.dart]:
```suggestion
      child: const Text('Reset password'),
```

Comment (stein155) [reply lib/src/config/login_options.dart]: I think this is the submit button so maybe 'Reset password' or 'Send reset email' is better here?

**Review by stein155 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**